images/bazel-tools/build.yaml

```diff
@@ -3,14 +3,18 @@ name: bazel-tools # Name of the image to be built
 variants:
   "10.24":
     arguments:
-      BASE_IMAGE: "eu.gcr.io/jetstack-build-infra-images/golang-dind@sha256:bf41f2a8f6191842ad3ff56a071448ed6a8bdcece4d6d62c5d91733c2f1f3541"
+      BASE_IMAGE: "eu.gcr.io/jetstack-build-infra-images/golang-dind@sha256:a71c5e6e8cb6875e9f6a43b722254693759b5db77566e25ba9eb52ca74b92e08"
+      # Version of Bazel that is bundled in the BASE_IMAGE
+      BAZEL_VERSION: "1.4.2"
```

Comment:
Should this be 4.2.1 ?

Reply:
Thanks for catching this Richard- it's a typo, the image does have the right version, I've just updated it

Yeah agree that it _can_  easily get incorrect (as we just saw), but I still would rather have the version number embedded in the tag. I think it's ok for whoever is adding this image tag to some code to still have to double check that everything works, but then at least the reader of the code doesn't have guess what versions of things are being used.

Reply:
Unfortunately we can't really check that the tags are correct since we use the same build script to make these images